markdown/bitburner.ns.mv.md

```diff
@@ -30,7 +30,7 @@ RAM cost: 0 GB
 
 Move the source file to the specified destination on the target server.
 
-This command only works for scripts and text files (.txt). It cannot, however, be used to convert from script to text file, or vice versa.
+This command only works for scripts and text files (.txt, .json). It cannot, however, be used to convert from script to text file, or vice versa.
```

Comment:
This is a little ambiguous, because it doesn't also list all the types of scripts. It might be better to not list any types here, since it distracts from the purpose.